### ISSUE TITLE
Flip / Mirror with BOTH vertical and horizontal parameters as true adds margin to image bounds

### ISSUE THREAD

**Issue by glenngartner** [oliver-moran/jimp#829]
# Expected Behavior

Calling flip() or mirror() on an image with both vertical and horizontal parameters as true should flip the image vertically and horizontally, without introducing artifacts to the flipped image.

# Current Behavior

Calling flip (or mirror) on an image with both vertical and horizontal parameters set to true adds a small white (2-3 pixels) margin around the border of the final image. 

# Failure Information (for bugs)

n/a -- see steps to reproduce

## Steps to Reproduce

Using this code snippet: 

``` ts
      Jimp.read('some/local/image/url/from/assets/dir').then(img => {
        const topLeft = img.clone().crop(0, 0, img.getWidth() / 2, img.getHeight() / 2).flip(true, true);
        const topLeft2 = img.clone().crop(0, 0, img.getWidth() / 2, img.getHeight() / 2).flip(true, false).flip(false, true);
}
```
See screenshots below. 

Left is the source image. I'm extracting quadrants from the image using crop, then flip each quadrant vertically and horizontally. 
The center image is topLeft (method above). 
The right-most image is topLeft2 (method above). 

I'm using a thick CSS border around each image to emphasize the white space issue I'm seeing. Calling the first method chain for "topLeft" introduces that white margin. Specifically, calling flip(true, true), results in the artifact. When calling flip(true, false).flip(false, true), I get the desired result, with no white "margin" around the image. 

I've used three separate images, and this occurs on each of them. 

**Screenshots**

![image](https://user-images.githubusercontent.com/10998599/70480307-625abe80-1aad-11ea-9e2a-e0663e6ba0bc.png)

## Context

- Jimp Version: "jimp": "^0.9.3", (browser version, typescript, with Angular 8
- Operating System: Windows 10
- Node version: 10.16.3

## Failure Logs
none

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in `v0.10.3` :rocket:

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in `v0.10.3` :rocket:
